src/OSM_dataTool.js

```diff
@@ -20,7 +19,7 @@ module.exports = function ({ routes, ways, assumeFirstWayIsStart, mapProperties,
             const data = routeExtractor(current_route, ways, assumeFirstWayIsStart)
 
             routes_complete++
-            log_file += `\nDone >>> ${name}`
+            log_file.push(Object.assign({ id: current_route.id }, current_route.tags))
```

Comment:
`id` might be overridden, if tags has an `id` key.

```suggestion
            log_file.push({ id: current_route.id, tags: current_route.tags })
```